projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -50,7 +50,6 @@ public function __construct( $connection_manager = null ) {
 			99
 		);
 		add_action( 'load-' . $page_suffix, array( $this, 'admin_init' ) );
-
```

Comment:
Did you mean to remove these lines of whitespace, or is it something that's set up in your editor?

Reply:
Some add_actions had a line break and I wrongly removed them, I have made it uniform now.